doc/JSON_FLAGS.md

```diff
@@ -1107,7 +1107,31 @@ These branches are also the valid entries for the categories of `dreams` in `dre
 - ```SIDEWALK``` Has sidewalks on the sides adjacent to roads.
 - ```LAKE``` Consider this location to be a valid lake terrain for mapgen purposes.
 - ```LAKE_SHORE``` Consider this location to be a valid lake shore terrain for mapgen purposes.
-
+- ```SOURCE_OF_FUEL``` For NPC AI, this location may contain fuel for looting.
```

Comment:
SOURCE_FUEL reads just as well and is slightly shorter.

Reply:
Ok changed them all to shorten them